### PR TITLE
nautilus: mgr/progress: delete all events over the wire

### PR DESCRIPTION
**Pull request by kamoltat**
Fixes an issue where the progress bar gets stuck when turning off the
module during rebalancing. Fixed by a cherry-picking commit from
master PR:
#33400

**Note: This issue only affects the nautilus branch, not the master.

Fixes: https://tracker.ceph.com/issues/48434

Signed-off-by: kamoltat <ksirivad@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/progress/module.py [hunk 1]
```diff
@@ -652,6 +652,7 @@ def clear(self):
         self._completed_events = []
         self._dirty = True
         self._save()
+        self.clear_all_progress_events()
```
Comment (neha-ojha) [inline src/pybind/mgr/progress/module.py:655]: @kamoltat aren't we effectively doing 6f60d33115d2f583331d31b95a0a33b96a614f09? If so, it is better to just cherry-pick the commit.

**Comment by smithfarm** [comment]
I agree with @neha-ojha - the preferred way forward here is to mark https://tracker.ceph.com/issues/48434 fixed by https://github.com/ceph/ceph/pull/33400 and turn this into a cherry-pick.

**Comment by yuriw** [comment]
[wip-yuri3-testing-2020-12-02-1604-nautilus](https://trello.com/c/z0lk7iqi/1077-wip-yuri3-testing-2020-12-02-1604-nautilus)

File: src/pybind/mgr/progress/module.py (hunk 1 above)
Comment (kamoltat) [reply src/pybind/mgr/progress/module.py:655]: I agree

**Comment by kamoltat** [comment]
PR Moved to #38416 I messed up on this PR

**Pull request closed without merging**